src/fetch/contracts.py

```diff
@@ -31,6 +33,7 @@ def __init__(
         self.network = network
         # de-duplicate to reduce unnecessary queries
         self.addresses = list(set(addresses))
+        self.null_balance_file = NetworkFile("null-balances.csv")
```

Comment:
nit: I don't understand see why this is an instance variable (it's only ever referenced in one method, so it could also just be created there?). I also noticed that the `contracts` methods takes a file as an argument whereas `get_null_balance` doesn't (no big deal but wondering if there is a reason for the different interface).

Reply:
The contracts method (as well as several other things in the project) would really benefit from some refactoring. We pass the file into the contracts method because this is a method with "lazy loading". since this null balance method doesn't load from file I didn't pass along a load from file. These two methods also differ in the sense that one is going to return the same results every time, but the other could change from one minute to the next.